app/assets/stylesheets/_variables.scss

```diff
@@ -26,29 +21,14 @@ $dark-gray: #333;
 $medium-gray: #999;
 $light-gray: #DDD;
 
-// Background Color
-$base-background-color: white;
```

Comment:
Have you used it any other time? Should that be just a variable for the button component? I removed this because I've used it so infrequently. I think in most cases it is making the code a little more complex unnecessarily.